### ISSUE TITLE
[BUG] [screen reader mode] Unable to input Japanese when selecting "Other" during tool: askUserQuestion, combined with screen reader mode

### ISSUE THREAD

**Issue by yncat** [anthropics/claude-code#81605]
### Preflight Checklist

- [x] I have searched [existing issues](https://github.com/anthropics/claude-code/issues?q=is%3Aissue%20state%3Aopen%20label%3Abug) and this hasn't been reported yet
- [x] This is a single bug report (please file separate reports for different bugs)
- [x] I am using the latest version of Claude Code

### What's Wrong?

Claude Code does not allow me to input Japanese when selecting "other" during a question tool call.

### What Should Happen?

Japanese characters are accepted.

### Error Messages/Logs

```shell

```

### Steps to Reproduce

1. Have Claude Code to ask me some questions using the question tool.
2. Choose "other" by inputting the last number in the selection.
3. try to input Japanese by typing or pasting.
4. Japanese characters are ignored.

Conditions:
axScreenReader must be on to reproduce this bug. It does not occur without it.

### Claude Model

Opus

### Is this a regression?

No, this never worked

### Claude Code Version

2.1.220 (Claude Code)

### Platform

Anthropic API

### Operating System

Windows

### Terminal/Shell

WSL (Windows Subsystem for Linux)

### Additional Information

I was able to reproduce this issue using the default WSL terminal, and it also occurs with my usual terminal (TeraTerm).
